extensions/amp-story/1.0/amp-story-tooltip.css

```diff
@@ -0,0 +1,152 @@
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS-IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+.i-amphtml-story-tooltip-layer {
+  top: 0 !important;
+  left: 0 !important;
+  width: 100% !important;
+  height: 100% !important;
+  z-index: 100001 !important;
+  position: absolute !important;
+}
+
+.i-amphtml-story-tooltip-layer::before {
+  content: '' !important;
+  position: absolute !important;
+  top: 0 !important;
+  left: 0 !important;
+  width: 100% !important;
+  height: 100% !important;
+  background-color: black !important;
+  opacity: 0.3 !important;
+  transition: opacity 0.15s cubic-bezier(0.4, 0.0, 1, 1) !important;
+}
+
+.i-amphtml-story-tooltip-layer.i-amphtml-hidden,
+.i-amphtml-story-tooltip-layer.i-amphtml-hidden::before {
+  opacity: 0 !important;
+  pointer-events: none !important;
+}
+
+.i-amphtml-story-tooltip-layer-nav-button-container {
+  height: 100% !important;
+  width: 160px !important;
+  position: absolute !important;
+}
+
+.i-amphtml-story-tooltip-layer-nav-button-container.i-amphtml-story-tooltip-nav-button-left {
+  background-image: linear-gradient(90deg, rgba(33,33,33,0.30) 2%, rgba(33,33,33,0) 100%) !important;
+  left: 0 !important;
+}
+
+.i-amphtml-story-tooltip-layer-nav-button-container.i-amphtml-story-tooltip-nav-button-right {
+  background-image: linear-gradient(90deg, rgba(33,33,33,0) 2%, rgba(33,33,33,0.30) 100%) !important;
+  right: 0 !important;
+}
+
+.i-amphtml-story-tooltip-layer-nav-button {
+  position: absolute !important;
+  top: 50% !important;
+  width: 48px !important;
+  height: 48px !important;
+  padding: 0 !important;
+  border: 0 !important;
+  background-color: transparent !important;
+  transition: transform 0.225s cubic-bezier(0.0, 0.0, 0.2, 1) !important;
+}
+
+.i-amphtml-story-tooltip-layer-nav-button.i-amphtml-story-tooltip-nav-button-left {
+  left: 0 !important;
+  background-image: url('data:image/svg+xml;charset=utf-8,<svg xmlns="http://www.w3.org/2000/svg" width="48px" height="48px" viewBox="0 0 48 48" fill="#FFFFFF"><path d="M30.83 14.83L28 12 16 24l12 12 2.83-2.83L21.66 24z"/><path d="M0 0h48v48H0z" fill="none"/></svg>') !important;
+}
+
+.i-amphtml-hidden .i-amphtml-story-tooltip-layer-nav-button.i-amphtml-story-tooltip-nav-button-left {
+  transform: translate3d(8px, 0, 0);
+}
+
+.i-amphtml-story-tooltip-layer-nav-button.i-amphtml-story-tooltip-nav-button-right {
+  right: 0 !important;
+  background-image: url('data:image/svg+xml;charset=utf-8,<svg xmlns="http://www.w3.org/2000/svg" width="48px" height="48px" viewBox="0 0 48 48" fill="#FFFFFF"><path d="M20 12l-2.83 2.83L26.34 24l-9.17 9.17L20 36l12-12z"/><path d="M0 0h48v48H0z" fill="none"/></svg>') !important;
```

Comment:
Can we reuse the `button-left` SVG and apply a `transform: rotate(180deg)`? This way you could apply the `background-image` to `nav-button` only once, saving some previous characters.

Reply:
We could work around that with an extra element but not sure it's worth it.